automated/linux/ota-apps/aklite-callback.sh

```diff
@@ -0,0 +1,4 @@
+#!/bin/bash -e
+
+echo "${MESSAGE}" >> /var/sota/ota.signal
+echo "${RESULT}" >> /var/sota/ota.result
```

Comment:
should this be '>>' or '>' ?
This file is the same across all ota-rollback, ota-update and ota-apps.
Can it be the same file?

Reply:
It can, and it would be preferred. Any ideas where to move this file? My original idea was that each test directory is self contained. Sharing files would break it, but in this instance it makes sense. Some suggestions are more than welcome.